Book/causalinference.Rmd

```diff
@@ -24,70 +24,70 @@ In this module, we introduce the counterfactual approach to causal inference and
 
  - [The introduction to experiments presentation from EGAP Learning Days in Guatemala City, Guatemala, August 2017](https://egap.github.io/learningdays-resources/Slides/Examples/intro_experiments-guatemala.pdf)
 
- - [The causal inference presentation from EGAP Learning Days in Salima, Malawi, February 2017](https://egap.github.io/learningdays-resources/Slides/Examples/causality-malawi.pdf)
+ - [La présentation de l'inférence causale aux EGAP Learning Days in Salima, Malawi, February 2017](https://egap.github.io/learningdays-resources/Slides/Examples/causality-malawi.pdf)
 
- - [The causal inference presentation from EGAP Learning Days at Universidad Diego Portales in Santiago, Chile, May 2016](https://egap.github.io/learningdays-resources/Slides/Examples/causality-santiago.pdf)
+ - [La présentation de l'inférence causale aux EGAP Learning Days at Universidad Diego Portales in Santiago, Chile, May 2016](https://egap.github.io/learningdays-resources/Slides/Examples/causality-santiago.pdf)
 
 ## Resources
 
-### EGAP Methods Guides
+### Guide des méthodes EGAP
 
-- EGAP Methods Guide [10 Things You Need to Know about Causal Inference](https://egap.org/resource/10-things-to-know-about-causal-inference/)
+- Guide des méthodes EGAP [10 choses à savoir sur l'inférence causale](https://egap.org/resource/10-things-to-know-about-causal-inference/)
 
-- EGAP Methods Guide [10 Strategies for Figuring Out If X Caused Y](https://egap.org/resource/10-strategies-figuring-out-if-x-caused-y/)
+- Guide des méthodes EGAP [10 stratégies pour déterminer si X a causé Y](https://egap.org/resource/10-strategies-figuring-out-if-x-caused-y/)
 
-- EGAP Methods Guide [10 Things You Need to Know about Mechanisms](https://egap.org/resource/10-things-mechanisms/)
+- Guide des méthodes EGAP [10 choses à savoir sur les mécanismes](https://egap.org/resource/10-things-mechanisms/)
 
-- EGAP Methods Guide [10 Things to Know About External Validity](https://egap.org/resource/10-things-to-know-about-external-validity/)
+- Guide des méthodes EGAP [10 choses à savoir sur la validité externe](https://egap.org/resource/10-things-to-know-about-external-validity/)
 
-### Books, Chapters, and Articles {#causalinference-cites}
+### Livres, chapitres et articles {#causalinference-cites}
 
-#### Classics {#causalinference-classics}
+#### Classiques {#causalinference-classics}
 
-- [@fisher_design_1935]. Fisher introduces the idea of randomization and hypothesis testing as a way to learn about causal inference.
+- [@fisher_design_1935]. Fisher introduit l'idée de la randomisation et des tests d'hypothèses pour apprendre l'inférence causale.
 
-- [@rubin:1974]. Rubin introduces the idea of potential outcomes and links counterfactual conceptualizations of causality to statistical inference.
+- [@rubin:1974]. TODO Rubin introduces the idea of potential outcomes and links counterfactual conceptualizations of causality to statistical inference.
 
-#### Contemporary Overviews
+#### Revue actuelle
```

Comment:
this is more like the best resources these days on the topics (pas revue actuelle)